testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/events/AbstractEventsTest.java

```diff
@@ -23,14 +23,15 @@
 import org.keycloak.testsuite.client.resources.TestingResource;
 import org.keycloak.testsuite.util.RealmBuilder;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  *
  * @author Stan Silvert ssilvert@redhat.com (C) 2016 Red Hat Inc.
  */
-public abstract class AbstractEventsTest extends AbstractKeycloakTest {
+public abstract class AbstractEventsTest extends AbstractKeycloakTest implements Serializable {
```

Comment:
Is it needed to use `Serializable` here? It is quite strange to me as none of our other tests need this...

I guess it is because of `testingClient.server(REALM_NAME1).run` block? In that case it can be possibility to:
- Use the real operations (to simulate real events) instead of "faking" them with testingClient
- Use the `final` local variable. Something like for example this should work AFAIK (based on some other classes):
```
final String localRealmId = this.realmId;
        testingClient.server(REALM_NAME_1).run(session -> {
            RealmModel realm = session.realms().getRealm(myRealmId);
```

WDYT?

Reply:
I changed to use `session.getContext().getRealm()` and removed the `Serializable`. Thanks!